docs/zh_cn/user_guides/visualization_feature_map.md

```diff
@@ -0,0 +1,201 @@
+# wandb记录特征图可视化
+
+MMSegmentation 1.x 提供了Weights & Biases的后端支持，方便对项目代码结果的可视化和管理。
+
+## Wandb的配置
+
+安装 Weights & Biases 的过程可以按照 [官方安装指南](https://docs.wandb.ai/quickstart)，具体的步骤如下:
+
+```shell
+pip install wandb
+wandb login
+```
+
+在 `vis_backend` 中添加 `WandbVisBackend`。
+
+```python
+vis_backends=[dict(type='LocalVisBackend'),
+              dict(type='TensorboardVisBackend'),
+              dict(type='WandbVisBackend')]
+```
+
+## 测试数据和结果及特征图的可视化
+
+`SegLocalVisualizer`是继承自 MMEngine 中`Visualizer` 类的子类，适用于 MMSegmentation 可视化，有关`Visualizer`的详细信息请参考在 MMEngine 中的[可视化教程](https://mmengine.readthedocs.io/zh_CN/latest/advanced_tutorials/visualization.html) 。
+
+以下是一个关于 `SegLocalVisualizer` 的示例，首先你可以使用下面的命令下载这个案例中的数据：
+
+<div align=center>
+<img src="https://user-images.githubusercontent.com/24582831/189833109-eddad58f-f777-4fc0-b98a-6bd429143b06.png" width="70%"/>
+</div>
+
+```shell
+wget https://user-images.githubusercontent.com/24582831/189833109-eddad58f-f777-4fc0-b98a-6bd429143b06.png --output-document aachen_000000_000019_leftImg8bit.png
+wget https://user-images.githubusercontent.com/24582831/189833143-15f60f8a-4d1e-4cbb-a6e7-5e2233869fac.png --output-document aachen_000000_000019_gtFine_labelTrainIds.png
+
+wget https://download.openmmlab.com/mmsegmentation/v0.5/ann/ann_r50-d8_512x1024_40k_cityscapes/ann_r50-d8_512x1024_40k_cityscapes_20200605_095211-049fc292.pth
+
+```
+
+```python
+# Copyright (c) OpenMMLab. All rights reserved.
+from argparse import ArgumentParser
+from typing import Type
+
+import mmcv
+import torch
+import torch.nn as nn
+
+from mmengine.model import revert_sync_batchnorm
+from mmengine.structures import PixelData
+from mmseg.apis import inference_model, init_model
+from mmseg.structures import SegDataSample
+from mmseg.utils import register_all_modules
+from mmseg.visualization import SegLocalVisualizer
```

Comment:
hi,could you tell me how to install these package from 51 to 54,thanks!

Reply:
> hi,could you tell me how to install these package from 51 to 54,thanks!

These path would be automatically searched in your mmseg codebase. You just need to follow the guidance in this README, create a python file `feature_map_visual.py` and put it in this place `./mmsegmentation/feature_map_visual.py`.

Reply:
dev-1.x docs link
https://mmsegmentation.readthedocs.io/en/dev-1.x/get_started.html

Reply:
hi,I've try reinstall the whole envriment as this doc describe,now my mmcv version is 2.0.0rc4,and mmengine0.5.0,mmsegmentation1.0.0rc5.but 'vidualize_feature_map.py' line 106,'inference' line 119, my 'data_'has no atrribute '['inputs']'(use 'loadimagefromfile').also,in 'mmcv.transform.loadimagefromfile',there is no '['inputs']'. so can you tell me your mmcv version?

Reply:
<img width="588" alt="image" src="https://user-images.githubusercontent.com/81373517/217791346-a0eb8611-b5d7-44f7-8ec3-7a5ea1a81085.png">

Reply:
> hi,I've try reinstall the whole envriment as this doc describe,now my mmcv version is 2.0.0rc4,and mmengine0.5.0,mmsegmentation1.0.0rc5.but 'vidualize_feature_map.py' line 106,'inference' line 119, my 'data_'has no atrribute '['inputs']'(use 'loadimagefromfile').also,in 'mmcv.transform.loadimagefromfile',there is no '['inputs']'. so can you tell me your mmcv version?

I also use 2.0.0rc4 mmcv

Reply:
I didn't fix it,but still thank you.